Content.Server/Construction/AnchorableSystem.cs

```diff
@@ -104,6 +113,28 @@ private void OnAnchorComplete(EntityUid uid, AnchorableComponent component, TryA
             );
         }
 
+        private bool CheckUnstackable(EntityUid uid, EntityUid user, EntityCoordinates coordinates)
+        {
+            if (_tagSystem.HasTag(uid, "Unstackable"))
+            {
+                // Need to enforce the unstackable rules on anchoring also.
+                var condition = new NoUnstackableInTile();
```

Comment:
Please do not construct dummy classes to work around APIs, I let it slide last time but it needs removing this time.

Reply:
Oh, I thought I removed the requirement for that.